pom.xml

```
@@ -14,17 +14,19 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.0.3.RELEASE</version>
+        <version>2.6.1</version>
```

I think that it would be better to upgrade to the latest version of Spring Boot 2, which is 2.7.7. It is not much different from 2.6.1 and contains the latest bug fixes.
```suggestion
        <version>2.7.7</version>
```